synapse/storage/databases/main/event_federation.py

```diff
@@ -1057,7 +1057,7 @@ def _get_connected_batch_event_backfill_results_txn(
             INNER JOIN batch_events AS c
             ON i.next_batch_id = c.batch_id
             /* Get the depth of the batch start event from the events table */
-            INNER JOIN events AS e USING (event_id)
+            INNER JOIN events AS e ON c.event_id = e.event_id
```

Comment:
> Complement tests pass with *either* `c` or `i` so weren't much use in discerning the two :/.

For reference, backfilling from the insertion or batch event will work. But we already have the insertion event so there isn't a need to backfill it again.